modules/visual_script/editor/visual_script_editor.cpp

```diff
@@ -912,15 +911,9 @@ void VisualScriptEditor::_update_graph(int p_only_id) {
 						const Vector<String> options = pi.hint_string.split(",");
 						uint32_t v = value;
 						for (const String &option : options) {
-							uint32_t current_val;
-							Vector<String> text_split = option.split(":");
-							if (text_split.size() != -1) {
-								current_val = text_split[1].to_int();
-							} else {
-								current_val = 1 << i;
-							}
+							uint32_t current_val = option.get_slice(":", 1).to_int();
```

Comment:
What if there's only one slice?

Reply:
Then it's 0.
The previous code didn't handle it really well either; checking size for -1 makes no sense.